README.md

```diff
@@ -1,41 +1,23 @@
 # Container Storage Interface (CSI) driver for vSphere
 
-This repository provides tools and scripts for building and testing `csi-vsphere`.
-The driver is an **alpha** state and should not be used in production.
+This repository provides tools and scripts for building and testing the vSphere CSI provider. This driver is in a stable `GA` state and is suitable for production use. It currently requires vSphere 6.7 U3 or higher in order to operate.
 
-On Kubernetes, the CSI driver is for use in conjunction with the out of tree vSphere
-[CCM](https://github.com/kubernetes/cloud-provider-vsphere).
+The CSI driver, when used on Kubernetes, also requires the use of the out-of-tree vSphere Cloud Provider Interface [CPI](https://github.com/kubernetes/cloud-provider-vsphere).
 
-The driver has been tested with Kubernetes 1.13 and 1.14, but due to internal K8s API
-changes during that cycle, an upgrade path for using `csi-vsphere` while moving from
-1.13 to 1.14 has not been documented.
+The driver has been tested with, and is supported on, K8s 1.13 and above.
```

Comment:
I thought v1.13 wouldn't work cause CSIDriver is not part of internal APIs yet, should this be v1.14?

Reply:
yeah, this seems right to me. When the CNS code was merged in, we deleted the 1.13 manifests -- which created the CSIDriver CRD. we don't do that anymore, because it was in-tree in 1.14.